css/style.css

```diff
@@ -0,0 +1,420 @@
+/* General tags */
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.add-hr {
+  color: black;
+  border-top: 1px solid #666;
+  width: 5em;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* header styles */
+
+.header {
+  display: flex;
```

Comment:
- [ ] Please check where you used `display: flex` and delete them, make a class called `flex` and apply it to all of the 'flexed' elements, so you don't repeat this property.